nautobot/core/settings.py

```diff
@@ -76,7 +76,7 @@
 HTTP_PROXIES = None
 JOBS_ROOT = os.getenv("NAUTOBOT_JOBS_ROOT", os.path.join(NAUTOBOT_ROOT, "jobs").rstrip("/"))
 MAINTENANCE_MODE = False
-
+AVAILABLE_IP_PREFIX_TIMEOUT = int(os.getenv("NAUTOBOT_AVAILABLE_IP_PREFIX_TIMEOUT", "600"))
```

Comment:
1. Maybe something more generic like `REDIS_LOCK_TIMEOUT`, so that this can be used if we determine the need for other similar locks in the future?
2. Location in this file seems pretty arbitrary. Maybe move it down to near the caching/Celery configs instead? (We don't really have any other general Redis configs in this file that I see).

Reply:
Both makes sense.

Reply:
Done